jekyll/_cci2/using-linuxvm.md

```diff
@@ -33,11 +28,29 @@ jobs:
 ```yaml
 version: 2.1
 jobs:
-  build:
+  my-job:
     machine: true
+    resource_class: large
 ```
 
-You can view the list of available images [here]({{ site.baseurl }}/2.0/configuration-reference/#available-linux-machine-images).
+You can view the list of available images [in the docs Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#available-linux-machine-images), or on the [Developer Hub](https://circleci.com/developer/images?imageType=machine). If you are working on an installation of CircleCI server, you will notice in the example above the syntax is slightly different, and the available Linux images are managed by your system administrator.
+
+## Available LinuxVM resource classes
+{: #available-linuxvm-resource-classes } 
+
+{% include snippets/machine-resource-table.md %}
+
+## Pre-installed software
+{: #pre-installed-software }
+
+The most up to date list of pre-installed software can be found on the [image builder](https://raw.githubusercontent.com/circleci/image-builder/picard-vm-image/provision.sh) page. You can also visit the [Discuss](https://discuss.circleci.com/) page for more information.
+
+Additional packages can be installed with `sudo apt-get install <package>`. If the package in question is not found, `sudo apt-get update` may be required before installing it.
+
+## Use machine with Docker
+{:  #use-machine-with-docker }
+
+Using the `machine` executor also means that you get full access to the Docker process. This allows you to run privileged Docker containers and build new Docker images.
```

Comment:
Same question as above - wasn't sure if we'd landed anywhere specifically yet on when to use `machine`.